docs/kb/semgrep-ci/why-duplicate-findings.md

```diff
@@ -0,0 +1,22 @@
+---
+description: To prevent duplicated findings, perform full scans only on the main branch of your repository.
+tags:
+  - Semgrep CI
+  - Semgrep Cloud Platform
+---
+
+# Why are duplicate findings appearing after running Semgrep in continuous integration (CI)?
+
+When scanning with Semgrep in CI, there are two types of scans you can perform: full scans and [**diff-aware scans**](/docs/semgrep-ci/running-semgrep-ci-with-semgrep-cloud-platform/#diff-aware-scanning).
+
+For full scans, the same rule and code produces a finding for every branch it is found on. If you are performing full scans on all branches, [the same finding appears for each branch](/docs/semgrep-code/findings/#deduplicating-findings).
+
+To prevent duplication, Semgrep recommends performing full scans only on the main branch of your repository, and performing diff-aware scans on PRs/MRs or other branches. Diff-aware scans compare findings on the current Git ref to findings on the base branch, allowing deduplication of findings not introduced in the PR/MR or branch.
```

Comment:
```suggestion
To prevent duplication, Semgrep recommends performing full scans only on the main branch of your repository and performing diff-aware scans on other branches in PRs or MRs. Diff-aware scans compare findings on the current Git ref to findings on the base branch, allowing deduplication of findings not introduced in the PR/MR branch.
```